### ISSUE TITLE
Calling `purs` needs to try some more candidates

### ISSUE THREAD

**Issue by justinwoo** [purescript/psc-package#138]
On Windows, purs.cmd, etc

**Comment by kritzcreek** [comment]
Just use shell expansion when calling the process

**Comment by taiju** [comment]
`npm-psc-package-bin-simple` also has a same issue as #34.
Probably, this issue is in <del>Turtle library</del><ins>System.Process.createProcess</ins>. I reproduced it easy with ghci.

```powershell
PS C:\Users\taiju> stack ghci

Warning: No local targets specified, so ghci will not use any options from your package.yaml /
         *.cabal files.

         Potential ways to resolve this:
         * If you want to use the package.yaml / *.cabal package in the current directory, use
           stack init to create a new stack.yaml.
         * Add to the 'packages' field of C:\sr\global-project\stack.yaml

Configuring GHCi with the following packages:
GHCi, version 8.0.2: http://www.haskell.org/ghc/  :? for help
Loaded GHCi configuration from C:\Users\taiju\AppData\Local\Temp\ghci16748\ghci-script
Prelude> import System.Process
Prelude System.Process> createProcess $ proc "purs" ["--version"]
*** Exception: purs: createProcess: does not exist (No such file or directory)
Prelude System.Process>
```

**Comment by taiju** [comment]
My workaround.

1. <del>Download purs for windows binary from https://github.com/purescript/purescript/releases</del> Copy `/path/to/node_modules/purescript/purs.bin` to `purs.exe`
2. Put it where there is purs.cmd. (e.g. `C:\Users\you\AppData\Roaming\npm`)

**Comment by justinwoo** [comment]
Fixed by #141
